src/test/rustdoc/issue-83832.rs

```diff
@@ -0,0 +1,21 @@
+#![crate_name = "foo"]
+
+pub mod io {
+    #[deprecated(since = "0.1.8", note = "Use bar() instead")]
+    pub trait Reader {}
+    pub trait Writer {}
+}
+
+// @has foo/mod1/index.html
+pub mod mod1 {
+    // @has foo/mod1/index.html '//code' 'pub use io::Reader;'
```

Comment:
```suggestion
    // @has - '//code' 'pub use io::Reader;'
```
Nit: You can use - to represent the previous mentioned file, the comments inside the modules could use it, but this isn't necessary.